data/mods/Magiclysm/enchanted/enchanted_rings.json

```
@@ -0,0 +1,328 @@
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "STRENGTH", "add": 4 } ] } ] }
+  },
+  {
+    "copy-from": "mring_copper",
+    "type": "TOOL_ARMOR",
+    "id": "mring_dexterity_1",
+    "name": "ring of dexterity +1",
+    "name_plural": "rings of dexterity +1",
+    "description": "A copper ring that makes you a little more agile when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "DEXTERITY", "add": 1 } ] } ] }
+  },
+  {
+    "copy-from": "mring_silver",
+    "type": "TOOL_ARMOR",
+    "id": "mring_dexterity_2",
+    "name": "ring of dexterity +2",
+    "name_plural": "rings of dexterity +2",
+    "description": "A silver ring that makes you a good bit more agile when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "DEXTERITY", "add": 2 } ] } ] }
+  },
+  {
+    "copy-from": "mring_gold",
+    "type": "TOOL_ARMOR",
+    "id": "mring_dexterity_3",
+    "name": "ring of dexterity +3",
+    "name_plural": "rings of dexterity +3",
+    "description": "A golden ring that makes you surprisingly more agile when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "DEXTERITY", "add": 3 } ] } ] }
+  },
+  {
+    "copy-from": "mring_platinum",
+    "type": "TOOL_ARMOR",
+    "id": "mring_dexterity_4",
+    "name": "ring of dexterity +4",
+    "name_plural": "rings of dexterity +4",
+    "description": "A platinum ring that makes you much more agile when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "DEXTERITY", "add": 4 } ] } ] }
+  },
+  {
+    "copy-from": "mring_copper",
+    "type": "TOOL_ARMOR",
+    "id": "mring_intelligence_1",
+    "name": "ring of intelligence +1",
+    "name_plural": "rings of intelligence +1",
+    "description": "A copper ring that makes you a little more intelligent when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "INTELLIGENCE", "add": 1 } ] } ] }
+  },
+  {
+    "copy-from": "mring_silver",
+    "type": "TOOL_ARMOR",
+    "id": "mring_intelligence_2",
+    "name": "ring of intelligence +2",
+    "name_plural": "rings of intelligence +2",
+    "description": "A silver ring that makes you a good bit more intelligent when you wear it.",
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "INTELLIGENCE", "add": 2 } ] } ] }
+  },
+  {
+    "copy-from": "mring_gold",
+    "type": "TOOL_ARMOR",
+    "id": "mring_intelligence_3",
+    "name": "ring of intelligence +3",
+    "name_plural": "rings of intelligence +3",
+    "description": "A golden ring that makes you a enlighteningly more intelligent when you wear it.",
```

```suggestion
    "description": "A golden ring that makes you surprisingly more intelligent when you wear it.",
```
to keep the symmetry, or
```suggestion
    "description": "A golden ring that makes you enlighteningly more intelligent when you wear it.",
```